src/chapters/razor-views/create-template.rst

```diff
@@ -0,0 +1,128 @@
+   of ``HelloASPDotNETDemo``. The final code presented in this 
+   video is found on the `views-static branch <https://github.com/LaunchCodeEducation/HelloASPDotNETDemo/tree/views-static>`__.
+
+The sections below outline the main ideas presented in the video. However, the
+text is NOT a substitute for completing the work described in the clip.
+
+.. index:: ! View() method
+
+Controllers Return View Templates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Out of the box, your MVC project contains several Razor templates within the ``Views`` directory.
+The sub-directory ``Home``, found inside of ``Views``, contains the Razor templates returned from the 
+``HomeController`` action methods. Just as an action method's name will map to a route with the same 
+name, an action method's name can also correspond to a Razor template's name. The ``Index()`` method 
+inside of ``HomeController`` returns the ``Index.cshtml`` template found inside of ``Views/Home``.
+
+In order to return that template, the action method calls a **View() method**. The ``View()`` method finds 
+a template that's associated with the particular controller and action method that it is called from. 
+
+To override the default behavior of the ``View()`` method, you can pass in a parameter which is the name 
+of the template you want to render. For example, if we want an action method named ``Form()`` to return a 
+Razor template named ``WelcomeForm.cshtml``, we have ``Form()`` return ``View("WelcomeForm");``.
+
+Add a Template
+^^^^^^^^^^^^^^
+
+To create a template, we update ``HelloController.Index()`` to return a Razor template instead of a 
+string of HTML. In ``Views``, create a new subdirectory called ``Hello``.
+
+Within ``Hello`` create a new file of the *Razor View* file template. 
+
+**WINDOWS USERS:** Right click on the ``Views`` directory or any subdirectory and select *Add -> New Item*. 
+Avoid selecting *Add -> View* from your options. 
+
+**MAC USERS:** Right click on the ``Views`` directory or any subdirectory and select *Add -> New File*.
+
+**ALL USERS:** In the modal window that appears, look for the item labelled *Razor View*. 
+
+.. figure:: figures/razor-view-template-selection.png
+   :alt: User selects the *Razor View* file template.
+
+   Select the *Razor View* file template.
+
+
+Add the form HTML to the template.
+Update the ``HelloController`` method ``Index()`` to return ``View()``. 
+
+When you re-render the app now, you'll notice some additional features and different styling.
+There is a ``_Layout.cshtml`` file inside of the ``Views/Shared`` folder that provides some scaffolding 
+for the application views. You can edit this file once and any other template will also render whatever 
+elements you add to it. This can help give your application a polished, unified look and feel.
+If you do not want to use this file, adding ``@{Layout = null;}`` at 
+the top of the template will cause this shared layout template to be ignored.
```

Comment:
I am not sure about this paragraph. I like where you are going with it and I think it drives home an important point! They just added the form to the template so now when they load the form page it is using the template which in turn uses ``_Layout.cshtml``. However, it took me a second to understand this point. My misunderstanding could be due to the fact that the app isn't running in front of me as I write this so I don't have the visual aide that the students will. However, from the student perspective, if I came back to this to simply review something from a later lesson, I would feel a little lost.

Reply:
yeah actually i feel the same way - i almost just left this in a comment to figure out when the rest of the chapter is fleshed out. 
I'll add a todo to return to this question later but keep the short intro here for now

Reply:
I think keeping this in will work! I reviewed the first video I made and the second one and I think my temptation after showing how you can "turn off" Layout to then bring it back was really why it felt so jumpy. I also like the intro to the idea of what is going on the ``Shared`` directory and why the files in there are important. I feel confident keeping the intro you have written here and adding that tidbit in the vid.